velox/exec/tests/utils/HiveConnectorTestBase.h

```diff
@@ -86,6 +86,29 @@ class HiveConnectorTestBase : public OperatorTestBase {
         remainingFilter);
   }
 
+  /// Build a HiveInsertTableHandle.
+  /// @param tableColumnNames Column names of the target table. Corresponding
+  /// type of tableColumnNames[i] is tableColumnTypes[i].
+  /// @param tableColumnTypes Column types of the target table. Corresponding
+  /// name of tableColumnTypes[i] is tableColumnNames[i].
+  /// @param partitionedBy A list of partition columns of the target table.
+  /// @param targetDirectory Final directory of the target table after commit.
+  /// @param writeDirectory Write directory of the target table before commit.
+  /// @param tableType Whether to create a new table, insert into an existing
+  /// table, or write a temporary table.
+  /// @param writeMode How to write to the target directory.
+  static std::shared_ptr<connector::hive::HiveInsertTableHandle>
+  buildHiveInsertTableHandle(
+      const std::vector<std::string>& tableColumnNames,
+      const std::vector<TypePtr>& tableColumnTypes,
+      const std::vector<std::string>& partitionedBy,
+      const std::string& targetDirectory,
+      const std::optional<std::string>& writeDirectory = std::nullopt,
```

Comment:
This method has a lot of parameters, but last 3 parameters are never used. Would it make sense to replace the last 4 parameters with a single LocationHandle parameter and provide convenience methods to create instances of LocationHandle.

Reply:
It's a good idea.